From 73450a56af5c5a8dcd7544a58d69f95cbe40d3f5 Mon Sep 17 00:00:00 2001
From: Tim Clephas <tim.clephas@nobleo.nl>
Date: Thu, 30 May 2024 14:27:46 +0200
Subject: [PATCH] Install package.xml

---
 CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 427e82e..c64ca05 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -56,7 +56,7 @@ option(dont_use_uploadtoblob "set dont_use_uploadtoblob to ON if the functionali
 # Compile options
 option(no_logging "disable logging" OFF)
 option(use_installed_dependencies "set use_installed_dependencies to ON to use installed packages instead of building dependencies from submodules" OFF)
-option(warnings_as_errors  "enable strict compiler warnings-as-errors" ON)
+option(warnings_as_errors  "enable strict compiler warnings-as-errors" OFF)
 option(strict_prototypes  "enable GCC strict-prototypes compiler option. This is not supported with test code enabled." OFF)
 option(build_as_dynamic "build the IoT SDK libaries as dynamic"  OFF)
 # Turn on/off IoT features
@@ -432,3 +432,6 @@ if (${use_installed_dependencies})
             ${package_location}
     )
 endif()
+
+# Install package.xml
+install(FILES package.xml DESTINATION share/azure_iot_sdk_c)
-- 
2.45.1

