diff --git a/CMakeLists.txt b/CMakeLists.txt
index b72bd41ed..40eb80a41 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -141,7 +141,7 @@ if (${use_azure_cloud_rsa_cert})
 endif()
 
 if (${use_azure_cloud_ecc_cert})
-    add_definitions(-DUSE_AZURE_CLOUD_ECC_CERT) 
+    add_definitions(-DUSE_AZURE_CLOUD_ECC_CERT)
 endif()
 
 if (${use_microsoftazure_de_cert})
@@ -419,3 +419,6 @@ if (${use_installed_dependencies})
             ${package_location}
     )
 endif()
+
++# Install catkin package.xml
++install(FILES package.xml DESTINATION share/azure-iot-sdk-c)
