From 17eb474cb05e449961c5aea5688a95782851e015 Mon Sep 17 00:00:00 2001
From: Tim Clephas <tim.clephas@nobleo.nl>
Date: Fri, 19 Jan 2024 08:14:48 +0100
Subject: [PATCH 2/2] Timple: Disable warnings

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3f20fda..faac024 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -56,7 +56,7 @@ option(dont_use_uploadtoblob "set dont_use_uploadtoblob to ON if the functionali
 # Compile options
 option(no_logging "disable logging" OFF)
 option(use_installed_dependencies "set use_installed_dependencies to ON to use installed packages instead of building dependencies from submodules" OFF)
-option(warnings_as_errors  "enable strict compiler warnings-as-errors" ON)
+option(warnings_as_errors  "enable strict compiler warnings-as-errors" OFF)
 option(strict_prototypes  "enable GCC strict-prototypes compiler option. This is not supported with test code enabled." OFF)
 option(build_as_dynamic "build the IoT SDK libaries as dynamic"  OFF)
 # Turn on/off IoT features
-- 
2.43.0

